deploy.php

```diff
@@ -0,0 +1,300 @@
+            continue;
+        }
+
+        // TODO should probably back up database
+
+        if (test('[ -f ' . $shortName . '/update.sql ]')) {
+            echo 'Running update.sql for ' . $shortName . "\n";
+
+            run('mysql --default-character-set="utf8" --host=' . $parameters['database.host'] . ' --port=' . $parameters['database.port'] . ' --user=' . $parameters['database.user'] . ' --password=' . $parameters['database.password'] . ' ' . $parameters['database.name'] . ' < ' . $shortName . '/update.sql');
+        }
+
+        if (test('[ -f ' . $shortName . '/locale.xml ]')) {
+            echo 'Installing locale.xml for ' . $shortName . "\n";
+
+            run('{{bin/php}} {{bin/console}} forkcms:locale:import -f ' . $dir . '/locale.xml --env={{symfony_env}}');
+        }
+
+        run('echo ' . $shortName . ' | tee -a {{deploy_path}}/shared/executed_migrations');
+    }
+})->desc('Migrate database and locale');
+
+task(
+    'fork:cache:clear',
+    function() {
+        run('{{bin/php}} {{bin/console}} fork:cache:clear --env={{symfony_env}}');
+        run('if [ -f {{deploy_path}}/shared/config/parameters.yml ]; then touch {{deploy_path}}/shared/config/parameters.yml; fi');
+    }
+)
+    ->desc('Clear Fork CMS cache');
+before('deploy:cache:clear', 'fork:cache:clear');
+
+task(
+    'sumo:symlink:document-root',
+    function() {
+        if (!get('document_root', false)) {
+            return;
+        }
+
+        $publicPath = get('deploy_path') . '/current/';
+        $currentSymlink = run(
+            'if [ -L {{document_root}} ]; then readlink {{document_root}}; fi'
+        );
+
+        $expandedPath = str_replace(
+            '~/',
+            run('echo $HOME') . '/',
+            $publicPath
+        );
+
+        // already linked, so we can stop here
+        if ($currentSymlink === $expandedPath) {
+            return;
+        }
+
+        // Show a warning when the document root exists. So we don't overwrite
+        // existing stuff.
+        if ($currentSymlink === '' && test('[ -e {{document_root}} ]')) {
+            writeln(
+                [
+                    '<comment>Document root already exists</comment>',
+                    '<comment>To link it, issue the following command:</comment>',
+                    sprintf(
+                        '<comment>ln -sf %1$s %2$s</comment>',
+                        $publicPath,
+                        get('document_root')
+                    ),
+                ]
+            );
+            return;
+        }
+
+        run(sprintf('mkdir -p %1$s', dirname(get('document_root'))));
+        run('rm -f {{document_root}}');
+        run(sprintf('{{bin/symlink}} %1$s {{document_root}}', $publicPath));
+    }
+)
+    ->desc('Symlink the document root to the public folder');
+after('deploy:symlink', 'sumo:symlink:document-root');
+
+desc('Enable a redirect page, all traffic will be redirected to this page.');
+task(
+    'sumo:redirect:enable',
```

Comment:
Niet logischer om https://github.com/tijsverkoyen/deployer-sumo in te laden? Dan is dat al aanwezig, zie https://github.com/tijsverkoyen/deployer-sumo/blob/master/recipe/redirect.php#L7